extra-ocaml/ocaml-pp/autobuild/build

```diff
@@ -0,0 +1,20 @@
+# Using AB3's build/15-dune.sh will install unneeded integrated test component,
```

Comment:
Doesn't make sense - review ABTYPE code to see what differences resulted in this issue.

Reply:
Thing is, `dune build` will build all available component, including those unneeded one for integrated testing purpose.

Reply:
This is not a AB3 fault but a behavior of dune itself